pyramid/scripts/pshell.py

```diff
@@ -159,18 +162,33 @@ def run(self, shell=None):
             closer()
 
     def make_shell(self):
+        shells = {}
+
+        priority_order = ['ipython', 'bpython']
```

Comment:
Only reason I have priority order here is for backwards compatibility will the fall through.